README.md

```diff
@@ -32,10 +32,12 @@ python main.py --port=8000
 Then the runtime parameters should be updated to:
 
 ```
-$ pip install splunk-opentelemetry
+$ sudo apt-get -y update
+$ pip3 install splunk-opentelemetry
+$ export PATH="$HOME/.local/bin:$PATH"
```

Comment:
This is very local and does not apply to all system. It highly depends on how and where pip is installed and if it is on path or not. Adding this here will imply we require `.local/bin` to contain pip which is just one of the dozens of locations it could be found at. 

I don't think we should mention it at all. People who have working python projects already know which pip and which python to execute for their projects.

Similar comment on `python` vs `python3`. Using python3 is fine but I think we might just add a line above mentioning this library is meant for python3 applications only. That should be good enough IMO.